src/resources/2014/CrimboShrub.ts

```diff
@@ -0,0 +1,85 @@
+import { toInt, visitUrl } from "kolmafia";
+import { have as have_ } from "../../lib";
+import { get } from "../../property";
+import { $familiar, $item } from "../../template-string";
+
+export function have(): boolean {
+  return have_($familiar`Crimbo Shrub`);
+}
+
+type Topper = "Muscle" | "Mysticality" | "Moxie";
+type Lights = "Prismatic" | "Hot" | "Cold" | "Stench" | "Spooky" | "Sleaze";
+type Garland = "HP Regen" | "PvP" | "Blocking";
+type Gifts = "Yellow Ray" | "Red Ray" | "Gifts";
+
+function topperChoice(topper: Topper) {
```

Comment:
Instead of defining the types directly as union strings, and having these helper functions, instead do this:

```
const toppers = {
  "Muscle": 1;
  "Mysticality": 2;
  "Moxie": 3;
  } as const;
  type Topper = keyof typeof toppers;